dnas/sensemaker/zomes/coordinator/sensemaker/src/utils.rs

```diff
@@ -54,3 +57,63 @@ pub fn flatten_btree_map<K, V: Clone>(btree_map: BTreeMap<K, Vec<V>>) -> Vec<V>
         .flatten()
         .collect::<Vec<V>>()
 }
+
+// create all entries specified in the config
+pub fn create_entries_from_config(
+    config: RawSensemakerConfig,
+    ca_key: AgentPubKey,
+) -> ExternResult<EntryHash> {
+    // dimensions
+    let _dimension_ehs = config
+        .dimensions
+        .clone()
+        .into_iter()
+        .map(|dimension: Dimension| create_entry(&EntryTypes::Dimension(dimension)))
```

Comment:
instead of using `create_entry` directly, it would be better to use `create_dimension` and the other `create_{sensemaker_type}` functions, so that if we want to change how these entries are indexed we only have to change it in one spot.

Reply:
Yes definitely. Thanks for pointing me to it. Will use that instead :D